### PR TITLE
LPS-155625 Give feedback to the user when they cannot save a fragment composition

### PR DESCRIPTION
**Pull request by DiegoHu97**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -10073,6 +10076,7 @@ name-identifier-attribute-type=Name Identifier Attribute Type
 name-identifier-format=Name Identifier Format
 name-is-required=Name is required.
 name-must-not-be-empty=Name must not be empty.
+name-must-not-be-null=Name must not be null
```
Comment (ruben-pulido) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10079]: Sentences need to end with a full stop. See "name-must-not-be-empty"

**Comment by DiegoHu97** [comment]
Resent here: https://github.com/liferay-echo/liferay-portal/pull/9079

**Pull request closed without merging**